lib/client/ai_client.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:async';
+
+import 'package:app/core/config.dart';
+import 'package:app/core/oauth/oauth.dart';
+import 'package:app/injection/register_module.dart';
+import 'package:ferry/ferry.dart';
+import 'package:ferry_hive_store/ferry_hive_store.dart' as ferry_hive_store;
+import 'package:graphql_flutter/graphql_flutter.dart';
+import 'package:hive_flutter/adapters.dart';
+import 'package:injectable/injectable.dart';
+
+@LazySingleton()
+class AIClient {
```

Comment:
can we put this file in gql folder ? cause this relates to gql stuff, no need to create another folder name client 
gql/ai_gql_client.dart should be fine